Formula/rex.rb

```diff
@@ -217,22 +232,23 @@ def install
 
     perl_build
     (libexec/"lib").install "blib/lib/Rex", "blib/lib/Rex.pm"
-
-    # Use system Perl
-    inreplace("bin/rexify") { |s| s.gsub!(/^#!perl$/, "#!/usr/bin/env perl") }
+    inreplace "bin/rex", "#!/usr/bin/env perl", "#!/usr/bin/perl"
+    inreplace "bin/rexify", "#!perl", "#!/usr/bin/perl"
 
     %w[rex rexify].each do |cmd|
       libexec.install "bin/#{cmd}"
-      chmod 0755, libexec/cmd
+      chmod 0555, libexec/cmd
       (bin/cmd).write_env_script(libexec/cmd, :PERL5LIB => ENV["PERL5LIB"])
       man1.install "blib/man1/#{cmd}.1"
     end
   end
 
   test do
-    assert_match "\(R\)\?ex #{version}", shell_output("#{bin}/rex -v"), "rex -v is expected to print out Rex version"
+    assert_match "\(R\)\?ex #{version}", shell_output("#{bin}/rex -v"),
```

Comment:
If it's not flagged by `brew audit`/`brew style` it's not worth changing in a random revision bump.